src/components/fields/SchemaField.js

```diff
@@ -88,8 +87,7 @@ function LabelInput(props) {
 function Help(props) {
   const { help } = props;
   if (!help) {
-    // See #312: Ensure compatibility with old versions of React.
-    return <div />;
+    return null;
```

Comment:
Can you make these changes (returning `null` instead of `<div>`'s) in a separate PR?

Reply:
`returning null instead of <div>'s` PR is already merged. I dont know why this commits is showing here. Will remove that commits.

Reply:
A bit messed up with the reverting part. So creating new PR #1246. Please delete this PR if possible.